content/appendices.md

```diff
@@ -611,11 +611,20 @@ characters permitted in user ID localparts. There are currently active
 users whose user IDs do not conform to the permitted character set, and
 a number of rooms whose history includes events with a `sender` which
 does not conform. In order to handle these rooms successfully, clients
-and servers MUST accept user IDs with localparts from the expanded
-character set:
+and servers MUST accept user IDs with localparts consisting of any legal
+unicode codepoint except for `:`, including zero characters. Localparts
+MUST be valid UTF-8 sequences.
```

Comment:
to be clear: we allow control chars (U+0000, etc) here?

Reply:
NUL is probably a special case, but other control characters are definitely allowed (I've created `@\x07:maunium.net`, `@\x08:maunium.net` and `@\x0B:maunium.net` at least)